Assets/HoloToolkit/Input/Scripts/SelectedObjectMessageSender.cs

```diff
@@ -0,0 +1,46 @@
+﻿using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// SelectedObjectMessageSender class sends a Unity message to currently selected object.
+/// Object selection is controlled via OnSelectObject and OnClearSelection events.
+/// Object selection and messages can be triggered using keyword responses registered in KeywordManager.
+/// </summary>
+public class SelectedObjectMessageSender : MonoBehaviour
```

Comment:
Yeah if multiple can trigger then then showing both examples would be nice.
Or stating that in the class description and readme.